tests/envcheck/testdata/go-0.1.0.textproto

```diff
@@ -0,0 +1,1963 @@
+name: "go-0.1.0"
```

Comment:
DId you want to check the handling for go-0.2.0 instead, or is this in reference to the spec version?

Reply:
It's a reference to the checker version, and we know that the go-0.2.0 checker has the same environment as go-0.1.0.